solr/core/src/java/org/apache/solr/parser/QueryParser.java

```diff
@@ -8,25 +8,25 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
-
 import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.queryparser.charstream.CharStream;
+import org.apache.lucene.queryparser.charstream.FastCharStream;
 import org.apache.lucene.search.BooleanClause;
 import org.apache.lucene.search.Query;
-import org.apache.solr.search.SyntaxError;
 import org.apache.solr.search.QParser;
-
-import org.apache.lucene.queryparser.charstream.CharStream;
-import org.apache.lucene.queryparser.charstream.FastCharStream;
+import org.apache.solr.search.SyntaxError;
 
 public class QueryParser extends SolrQueryParserBase implements QueryParserConstants {
```

Comment:
Yea I did format it. If it were in like `.generated.` or something it would be easy to exclude. I didn't see a downside to formatting the file even though it was machine generated.

Reply:
Okay.  Since Lucene has the same concern, we should probably mimick them.  See this commit: https://github.com/apache/lucene/commit/2cbf261032dc8aca56c846971c090c991ac594a6 notice that the spotless config expressly excluded these generated files.

Reply:
I don't really agree, but not worth arguing over. Fixed in fade0a75a06504e9d50ba21314fa2f35ff2a5532